paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationInterceptor.kt

```diff
@@ -177,8 +177,12 @@ internal class DefaultIntentConfirmationInterceptor @Inject constructor(
         shippingValues: ConfirmPaymentIntentParams.Shipping?,
         setupForFutureUsage: ConfirmPaymentIntentParams.SetupFutureUsage?,
     ): NextStep {
+        var attribution = paymentMethodCreateParams.attribution + "deferred-intent"
+        if(intentConfiguration.paymentMethodTypes.isNullOrEmpty()){
```

Comment:
I think the build failure is related to spacing on this line. You can test locally by running `./gradlew detekt ktlint`